src/v2/views/board/NewContactGroupBoard.tsx

```diff
@@ -0,0 +1,352 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React, { useCallback, useContext, useMemo, useState } from 'react';
+
+import {
+	Container,
+	Button,
+	Text,
+	Input,
+	InputProps,
+	Avatar,
+	ListV2,
+	SnackbarManagerContext,
+	Row
+} from '@zextras/carbonio-design-system';
+import { useBoardHooks } from '@zextras/carbonio-shell-ui';
+import { remove, size, some, uniqBy } from 'lodash';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import { MemberListItemComponent } from '../../../components/member-list-item';
+import { CHIP_DISPLAY_NAME_VALUES } from '../../../constants/contact-input';
+import ContactInput from '../../../integrations/contact-input';
+import { CONTACT_GROUP_TITLE_MAX_LENGTH } from '../../constants';
+import { client } from '../../network/client';
+
+const List = styled(ListV2)`
+	min-height: 0;
+`;
+
+const NewContactGroupBoard = (): React.JSX.Element => {
+	const [t] = useTranslation();
+	const { updateBoard, closeBoard } = useBoardHooks();
+	const createSnackbar = useContext(SnackbarManagerContext);
+
+	const initialTitle = t('board.newContactGroup.title', 'New Group');
+	const [titleValue, setTitleValue] = useState(initialTitle);
+
+	const [contactInputValue, setContactInputValue] = useState<
+		Array<{
+			email: string;
+			error: boolean;
+			duplicated: boolean;
+			actions: Array<{ color: string; icon: string; id: string; type: string }>;
+		}>
```

Comment:
I think it is better to use the ChipItem.
```suggestion
		Array<ChipItem & {
			email: string;
			duplicated: boolean;
		}>
```